lalrpop-test/src/lib.rs

```diff
@@ -192,6 +192,9 @@ pub fn use_cfg_created_parser() {
     #[cfg(feature = "test-set")]
     cfg::CreatedParser::new();
     cfg::AlwaysCreatedParser::new();
+    cfg::CreatedWithNotParser::new();
+    cfg::CreatedWithAnyParser::new();
+    cfg::CreatedWithAllParser::new();
 }
```

Comment:
I like this approach to testing, if Rust resolves the cfg feature annotation then we should too.

This test does look incomplete however. Note what happens when you run `cargo test --no-default-features` since this test currently only works because of the default setting.